### PR TITLE
Isx 1883 docs project wide actions as asset

### PR DESCRIPTION
**Pull request by duckets**
### Description
https://jira.unity3d.com/browse/ISX-1883

ISX-1883
Documentation updates for storing project wide input actions as an asset rather than in inputmanager.asset

### Changes made

 updates to a large number of pages and screenshots

### CODE REVIEW

File: Packages/com.unity.inputsystem/Documentation~/Actions.md
Comment (lyndon-unity) [inline Packages/com.unity.inputsystem/Documentation~/Actions.md:78]: I think we could add something here about the benefits and drawbacks

E.g. This allows bundling of the control bindings and behaviour together in a single monobehaviour or prefab so if can be distributed together. But it makes finding the full list of control bindings more difficult as its distributed in multiple locations.

Comment (lyndon-unity) [reply Packages/com.unity.inputsystem/Documentation~/Actions.md:78]: The further up we should talk up the benefits of the project wide input actions.

E.g. All controls listed in a single location for ease of editing and viewing.

File: Packages/com.unity.inputsystem/Documentation~/ActionAssets.md
Comment (ekcoh) [inline Packages/com.unity.inputsystem/Documentation~/ActionAssets.md]: I just generated it, so its definitely possible via asset Inspect menu. Might this text be from before we switched back to asset-based?

Comment (ekcoh) [reply Packages/com.unity.inputsystem/Documentation~/ActionAssets.md]: Hence this should be described for Input Actions Asset in general, there is no different between assets since the option is only available from asset Inspector.

File: Packages/com.unity.inputsystem/Documentation~/ActionAssets.md
Comment (ekcoh) [inline Packages/com.unity.inputsystem/Documentation~/ActionAssets.md:120]: This is way better

File: Packages/com.unity.inputsystem/Documentation~/Actions.md
Comment (ekcoh) [inline Packages/com.unity.inputsystem/Documentation~/Actions.md]: Should this tie back to the project wide aspect? Its currently a bit generic, e.g. 
"A reference to the set of project-wide actions configured in...."

File: Packages/com.unity.inputsystem/Documentation~/Actions.md
Comment (ekcoh) [inline Packages/com.unity.inputsystem/Documentation~/Actions.md]: I would suggest to drop "developing something unusual or " from the last sentence, its just projecting an opinion upon the users vision. The sentence is good enough without that part.

File: Packages/com.unity.inputsystem/Documentation~/Actions.md
Comment (ekcoh) [inline Packages/com.unity.inputsystem/Documentation~/Actions.md]: This "embed actions" is fine I guess but very vague what it means. I would rather suggest, 
"When you embed actions by defining serialised InputAction fields in a MonoBehavior....".
Might point out that for this scenario, the serialised action configuration is stored with the parent object as part of the scene, opposite to being serialised with an InputActionAsset.

File: Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md
Comment (ekcoh) [inline Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md]: Why is gameplay capitalised in this sentence?

File: Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md
Comment (ekcoh) [inline Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md:9]: Could be more specific in this paragraph and mention that it is available as a "preloaded asset" which is loaded at player start-up and kept alive until the player terminates. See https://docs.unity3d.com/ScriptReference/PlayerSettings.GetPreloadedAssets.html

File: Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md
Comment (ekcoh) [inline Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md]: Looks like some word got lost in the last sentence here?

**Review by ekcoh — APPROVED** [review]
LGTM, awesome improvements @duckets. I left some comments about minor improvements and clarifications.

File: Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md
Comment (lyndon-unity) [inline Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md]: I'd be tempted to split action maps and control schemes onto separate lines as I think there is an opportunity to introduce the differences a little here:

Calling out the actions maps provide a way to group the actions for ease of enabling/disabling groups of actions. 
Control schemes provide a way to enable/disable a set of devices which cut across actions (e.g. enable just gamepad input across move, jump, crouch etc).

File: Packages/com.unity.inputsystem/Documentation~/Concepts.md
Comment (lyndon-unity) [inline Packages/com.unity.inputsystem/Documentation~/Concepts.md]: Could potentially mention in code via InputSystem.actions here

File: Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md
Comment (lyndon-unity) [inline Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md]: There is also a menu option to reset to defaults or to delete all maps - worth mentioning and showing a screenshot of where in UI

File: Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md
Comment (lyndon-unity) [inline Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md:58]: We should also mention that the project-wide input actions are enabled by default.

**Review by lyndon-unity — APPROVED** [review]
Good step forward - some additional changes could improve further but can lad as is and enhance more.

File: Packages/com.unity.inputsystem/Documentation~/ActionAssets.md
Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/ActionAssets.md]: Yes this is left-over from when you couldn't do this with project-wide assets

Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/ActionAssets.md]: fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)

File: Packages/com.unity.inputsystem/Documentation~/ActionAssets.md
Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/ActionAssets.md:120]: :)

File: Packages/com.unity.inputsystem/Documentation~/Actions.md
Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/Actions.md]: fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)

File: Packages/com.unity.inputsystem/Documentation~/Actions.md
Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/Actions.md]: fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)

File: Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md
Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md]: fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)

File: Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md
Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md]: fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)

File: Packages/com.unity.inputsystem/Documentation~/Actions.md
Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/Actions.md:78]: fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)

File: Packages/com.unity.inputsystem/Documentation~/Concepts.md
Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/Concepts.md]: fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)

File: Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md
Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md:58]: fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)

File: Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md
Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md]: fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)

File: Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md
Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md:9]: fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)

File: Packages/com.unity.inputsystem/Documentation~/Actions.md
Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/Actions.md]: fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)

File: Packages/com.unity.inputsystem/Documentation~/Actions.md
Comment (lyndon-unity) [inline Packages/com.unity.inputsystem/Documentation~/Actions.md]: That link looks potentially wrong - should be project wide actions - not processors

Comment (lyndon-unity) [reply Packages/com.unity.inputsystem/Documentation~/Actions.md]: ProjectWideActions.md

**Review by lyndon-unity — APPROVED** [review]
One link looks incorrect. Approving with that changed.

File: Packages/com.unity.inputsystem/Documentation~/Actions.md
Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/Actions.md]: fixed in https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/0c5b32bba250166b2b85754d06e03c25b2d3b6fd

File: Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md
Comment (duckets) [reply Packages/com.unity.inputsystem/Documentation~/ProjectWideActions.md]: added in https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/2900746318b87ed4599daef9f0bf71490114d749

File: Packages/com.unity.inputsystem/Documentation~/Actions.md
Comment (lyndon-unity) [reply Packages/com.unity.inputsystem/Documentation~/Actions.md]: I see update fixes this

File: Packages/com.unity.inputsystem/Documentation~/PreReleaseNotes.md
Comment (lyndon-unity) [inline Packages/com.unity.inputsystem/Documentation~/PreReleaseNotes.md:1]: Some links are broken since this is removed

"PreReleaseNotes.md in C:\build\output\Unity-Technologies\InputSystem\Temp\docstest\com.unity.inputsystem@1.8\manual\TableOfContents.html (File Not Found)", "PreReleaseNotes.md in C:\build\output\Unity-Technologies\InputSystem\Temp\docstest\com.unity.inputsystem@1.8\manual\toc.html (File Not Found)"

File: Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md
Comment (lyndon-unity) [inline Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md:15]: Some links are broken since this rename

"ActionsEditor.html#the-default-actions in C:\build\output\Unity-Technologies\InputSystem\Temp\docstest\com.unity.inputsystem@1.8\manual\RespondingToActions.html (Tag Not Found)", "ActionsEditor.html#the-default-actions in C:\build\output\Unity-Technologies\InputSystem\Temp\docstest\com.unity.inputsystem@1.8\manual\RespondingToActions.html (Tag Not Found)",

**Review by lyndon-unity — APPROVED** [review]
Test fixes need making to resolve issues - see comments above on a couple of the issues.

Approved when this is done.

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/InputSystem?pullRequest=1867) <br/>All committers have signed the CLA.

**Pull request merged**
